internal/command/delta.go

```diff
@@ -70,14 +76,24 @@ func delta(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("preparing new deployment: %w", err)
 	}
 
-	// Create a new deployment delta
-	//
-	log.Print("Creating a new deployment delta...\n")
 	client, err := api.NewClient(apiUrl, apiToken, http.DefaultClient)
 	if err != nil {
 		return err
 	}
-	res, err := client.CreateDelta(context.Background(), orgID, appID, delta)
+
+	var res *ht.DeploymentDelta
+	if deltaID == "" {
+
+		log.Print("Creating a new deployment delta...\n")
+		res, err = client.CreateDelta(cmd.Context(), orgID, appID, delta)
+
+	} else {
+
+		log.Printf("Updating existing delta %s in place...\n", deltaID)
+		updates := []*ht.UpdateDeploymentDeltaRequest{{Modules: delta.Modules}}
```

Comment:
There could also be updates to `shared`

Reply:
aha yes I missed that because the api docs were missing that in the example. fixed now 👍